cool-core/src/test/java/com/nus/cool/core/resources/parsertest/table.csv

```diff
@@ -0,0 +1,4 @@
+id,birthYear,event,attr1,attr2,attr3,attr4,time
```

Comment:
I think you can use the csv table which we have created before.
No need to create another mini dataset

Reply:
agree, thanks